wiki/Main_Page/es.md

```diff
@@ -114,11 +111,11 @@ Organizaciones: [osu! UCI](/wiki/Community/Organisations/osu!_UCI)
 
 # [Personas](/wiki/People)
 
-osu! no hubiera sido posible sin tantos usuarios ayudando con el desarrollo, mantenimiento, y gestión de la comunidad: sus esfuerzos, tiempo, y dedicación hacen que el juego viva y prospere.
+osu! no hubiera sido posible sin la ayuda de muchos usuarios con el desarrollo, el mantenimiento y la gestión de la comunidad: su esfuerzo, tiempo y dedicación hacen que el juego viva y prospere.
 
-[El Equipo](/wiki/People/The_Team): [Desarrolladores](/wiki/People/The_Team/Developers) • [Equipo de Moderación Global](/wiki/People/The_Team/Global_Moderation_Team) • [Equipo de Soporte](/wiki/People/The_Team/Support_Team) • [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) • [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) • [osu! Alumni](/wiki/People/The_Team/osu!_Alumni) • [Equipo del Proyecto Amado](/wiki/People/The_Team/Project_Loved_Team)
+[El equipo](/wiki/People/The_Team): [Desarrolladores](/wiki/People/The_Team/Developers) • [Equipo de Moderación Global](/wiki/People/The_Team/Global_Moderation_Team) • [Equipo de Soporte](/wiki/People/The_Team/Support_Team) • [Nomination Assessment Team](/wiki/People/The_Team/Nomination_Assessment_Team) • [Beatmap Nominators](/wiki/People/The_Team/Beatmap_Nominators) • [osu! Alumni](/wiki/People/The_Team/osu!_Alumni) • [Equipo del Proyecto Amado](/wiki/People/The_Team/Project_Loved_Team) • [Beatmap Spotlight Curators](/wiki/People/The_Team/Beatmap_Spotlight_Curators)
```

Comment:
Same thing with project loved here.

I think having `Equipo` capitalized is correct here, similarly to what was pointed out earlier with project loved.